src/libraries/Microsoft.Extensions.DependencyModel/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
```

Comment:
Do we just want to follow the Assembly version that the rest of libraries are using(5.0.0.0)? Or do we want to override the default to match what the installer had before?

Reply:
The assembly version of the "core-setup" managed libraries has followed the versioning of the .NET Core product as a whole - the same as the `dotnet.exe` host.

The 3.1.3 package has:

![image](https://user-images.githubusercontent.com/8291187/78177451-c3731d00-7423-11ea-9cfa-a19488af8a05.png)

(Why we bumped the patch AssemblyVersion, I don't know.)

The 5.0.0-preview.1.20120.5 package has:

![image](https://user-images.githubusercontent.com/8291187/78177538-e69dcc80-7423-11ea-8f60-4ca1be513428.png)

So continuing with `5.0.0.0` will work for this library.